client/app/components/game-client.js

```diff
@@ -0,0 +1,69 @@
+import Ember from 'ember';
+import GameChannel from 'vilive/utils/game-channel';
+
+export default Ember.Component.extend({
+  init() {
+    this.set('messages', Ember.makeArray());
+    GameChannel.init(this.get('session-manager').currentToken());
```

Comment:
Did some research, invalid token returns the message "error" and nothing else. I was looking at the UserSocket code and noticed we just return the atom :error, but appears that is not an additional form that allows messages to go back.

I think we need to revisit the security model at some point anyway as right now we only validate token on the socket connect, and if that connection happens before the expiration there are no other checks. So theoretically, I could open up the dev tools manually connect and keep that connection open indefinitely after the expiration.

Not sure how much it matters now, just something that comes to mind. I would think that having the server issue a disconnect when the user logs out (having the client hit a logout url) and on login doing the same will force the connect close so the expired token isn't valid anymore.

Possibly as we progress we can combat this as well by instead of using user_id we gen UUIDs so that if their is a rogue socket out there it would be using the wrong id after some point. Neither here nor there for now though.